### PR TITLE
Fix: Current cops, electronickit zone and success/error messages 

### PR DESCRIPTION
**Pull request by Tanges**
## Description

I added an event handler to get latest amount of cops on duty and store it server side for all clients to access it.
I fixed not being able to start the hack since you were never in the electronickit zone.
Removed check for station hit to interact with it, display error message instead that it has already been hit.

## Checklist

- [ x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by mafewtm** [comment]
You dont need to keep making new PRs every time you change something. Your new commits will go in here

**Comment by Tanges** [comment]
Oh, my bad. I'm pretty new to PRs but I will remember that from now on :)

File: client/fleeca.lua [hunk 1]
```diff
@@ -24,8 +24,9 @@ RegisterNetEvent('electronickit:UseElectronickit', function()
 
     local isBusy = lib.callback.await('qb-bankrobbery:server:isRobberyActive', false)
     if isBusy then return exports.qbx_core:Notify(Lang:t('error.security_lock_active'), 'error', 5500) end
-
-    if CurrentCops < config.minFleecaPolice then return exports.qbx_core:Notify(Lang:t('error.minimum_police_required', {police = config.minFleecaPolice}), 'error') end
+    
+    currentCops = exports.qbx_core:GetDutyCountType('leo')
```
Comment (mafewtm) [inline client/fleeca.lua:28]: This is a server-sided function

File: client/pacific.lua [hunk 2]
```diff
@@ -21,8 +22,9 @@ RegisterNetEvent('qb-bankrobbery:UseBankcardB', function()
 
     local isBusy = lib.callback.await('qb-bankrobbery:server:isRobberyActive', false)
     if isBusy then return exports.qbx_core:Notify(Lang:t('error.security_lock_active'), 'error', 5500) end
-
-    if CurrentCops < config.minPacificPolice then return exports.qbx_core:Notify(Lang:t('error.minimum_police_required', {police = config.minPacificPolice}), 'error') end
+    
+    currentCops = exports.qbx_core:GetDutyCountType('leo')
```
Comment (mafewtm) [inline client/pacific.lua:26]: Ditto

File: client/paleto.lua [hunk 3]
```diff
@@ -12,7 +13,8 @@ RegisterNetEvent('qb-bankrobbery:UseBankcardA', function()
     local isBusy = lib.callback.await('qb-bankrobbery:server:isRobberyActive', false)
     if isBusy then return exports.qbx_core:Notify(Lang:t('error.security_lock_active'), 'error', 5500) end
 
-    if CurrentCops < config.minPaletoPolice then return exports.qbx_core:Notify(Lang:t('error.minimum_police_required', {police = config.minPaletoPolice}), 'error') end
+    currentCops = exports.qbx_core:GetDutyCountType('leo')
```
Comment (mafewtm) [inline client/paleto.lua:16]: Ditto

File: client/powerstation.lua [hunk 4]
```diff
@@ -38,12 +39,13 @@ end)
 
 RegisterNetEvent('thermite:UseThermite', function()
     local pos = GetEntityCoords(cache.ped)
+    currentCops = exports.qbx_core:GetDutyCountType('leo')
```
Comment (mafewtm) [inline client/powerstation.lua:42]: Ditto

**Comment by mafewtm** [comment]
You also don't need `local currentCops = 0` in every file

**Pull request closed without merging**